inference-engine/tests/unit/gna/gna_api_stub.cpp

```diff
@@ -129,6 +129,9 @@ GNA2_API enum Gna2Status Gna2RequestEnqueue(
 GNA2_API enum Gna2Status Gna2RequestWait(
     uint32_t requestId,
     uint32_t timeoutMilliseconds) {
+    if (current != nullptr) {
+        return current->Gna2RequestWait(requestId, timeoutMilliseconds);
+    }
     return Gna2StatusSuccess;
```

Comment:
I know that this is the typical flow of the unit tests, but to me it seems a bit weird, shouldn't we throw here? (it means test is not unit testing since interface is not here)

Reply:
Do as you wish, since it is normal for this project you may leave it as is

Reply:
I will leave like that.
IMO it was designed like that, as `if (current == nullptr) the gtest's mocking framework is not enabled and some u-tests may NOT want to enable mocks. And those can enforce `current==nullptr`.